proposals/2025-06-23-tsc-voting-procedures.md

```diff
@@ -0,0 +1,177 @@
+# TSC Voting Procedures Proposal
+
+## Summary
+
+This proposal establishes standardized voting procedures for the Hiero Technical Steering Committee (TSC) to improve decision-making efficiency while maintaining transparency and adherence to charter requirements.
+
+## Motivation
+
+Sometimes the TSC struggles to achieve qourum around key decisions or requires additional time outside of a meeting to deliberate on a decision. Asynchronous voting through Github Pull Requests would enable both flexibility and a solution for voting outside of the short meeting window.  
+
+## Proposal Details
+
+## Voting Methods
+
+### 1. In-Meeting Votes
+
+- **When Used**: For decisions during TSC meetings with quorum present
+- **Threshold**: Simple majority of members present
+- **Recording**: Documented in meeting minutes with vote counts
+
+### 2. Asynchronous Votes via GitHub Pull Request
+
+- **When Used**:
+  - When [quorum](https://github.com/hiero-ledger/hiero/blob/main/technical-charter.md?plain=1#L206) cannot be achieved in meetings
+  - When voting must be completed before next TSC meeting
+  - When a TSC member requests async voting due to inability to attend meeting
+  - For charter-mandated votes requiring 2/3 majority [per Charter section 4. iii](https://github.com/hiero-ledger/hiero/blob/main/technical-charter.md?plain=1#L209)
+- **Location**: Pull requests in the `proposals/` directory
+- **Notification**: All TSC members must be notified via:
+  - GitHub PR mention (@username)
+  - Slack notification in TSC channel
+
+## Voting Process for In-Meeting Votes
+
+### Step 1: Proposal Introduction
+- Present the proposal during TSC meeting
+- Allow for discussion and clarification
+
+### Step 2: Call for Vote
+- TSC Chair or designee calls for vote
+- Confirm [quorum](https://github.com/hiero-ledger/hiero/blob/main/technical-charter.md?plain=1#L206) is present
+
+### Step 3: Voting
+- Members vote verbally or via meeting platform
+- Record each member's vote
+
+### Step 4: Resolution
+- Tally votes and announce result
+- Document in meeting minutes with vote counts
+
+## Voting Process for Asynchronous Votes
+
+### Step 1: Discussion (Optional)
+
+- Create a GitHub Discussion for complex proposals requiring discussion
+- Label with "tsc-discussion"
+- No formal timeline - continue until consensus on proposal language
+
+### Step 2: Proposal Creation
+
+1. Create a new file in `proposals/` directory following the template
+2. File naming convention: `YYYY-MM-DD-brief-description.md`
```

Comment:
I'm not sure the date is going to be useful here. What if we had a proposal number, similar to HIP numbers, where the number is the PR number. Then we get some kind of sequence?